router/http/src/main/scala/io/buoyant/router/http/EnvTenantHostIdentifier.scala

```diff
@@ -38,7 +42,7 @@ case class EnvTenantHostIdentifier(
     Dst.Path(prefix ++ path, baseDtab(), Dtab.local)
 
   def apply(req: Request): Future[RequestIdentification[Request]] = {
-    val tenant = getHeader(req, TenantHeader)
+    val tenant = getHeader(req, TenantHeader).orElse(defaultTenant)
```

Comment:
do we need to do a check here and throw error if both TenantHeader and defaultTenant are not set?

Reply:
No, this is an Option (similar to Java's Optional). We throw below if it's empty.